app/components/join-steps/step-two.js

```diff
@@ -5,6 +5,13 @@ import { validator } from '../../utils/validator';
 import { debounce } from '@ember/runloop';
 import { JOIN_DEBOUNCE_TIME } from '../../constants/join';
 
+const LIMITS = {
+  introduction: 100,
+  skills: 1,
+  college: 1,
+  forFun: 100,
+  funFact: 100,
+};
```

Comment:
Please import this from constants file `constants/join.js` from which JOIN_DEBOUNCE_TIME is imported